src/docbuilder/metadata.rs

```diff
@@ -119,6 +135,41 @@ impl Metadata {
 
         metadata
     }
+    // Return (default_target, all other targets that should be built with duplicates removed)
+    pub(super) fn targets(&self) -> (&str, Vec<&str>) {
```

Comment:
It'd be better to have a struct as the return type, to make it clear that one is the default target and the other is the rest of the targets.